applications/h264/h264_endoscopy_tool_tracking_distributed/cpp/main.cpp

```diff
@@ -0,0 +1,126 @@
+/*
+ * SPDX-FileCopyrightText: Copyright (c) 2024 NVIDIA CORPORATION & AFFILIATES. All rights
+ * reserved. SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <getopt.h>
+#include <iostream>
+#include <string>
+
+#include <holoscan/holoscan.hpp>
+
+#include "cloud_inference_fragment.cc"
+#include "video_input_fragment.cc"
+#include "viz_fragment.cc"
```

Comment:
We can treat them as head-only libraries, so it might be better to rename them to `.hpp` files.